.dockerignore

```diff
@@ -0,0 +1,8 @@
+/prerelease.sh
+/README.md
+/INSTALL
+/LICENCE.TXT
+/CHANGES.md
+/.gitignore
+/.git
+/docs.tar.gz
```

Comment:
TL;DR - add a new line at the end of the file

Your editor needs setting to always include a new line at the end of a file.  Files are not technically text without this.

Reply:
Added all of the changes.